### PR TITLE
Allow underscore in Kinto resource names.

### PR DESCRIPTION
**Pull request by Natim**
r? @ametaireau

### CODE REVIEW

File: kinto/views/__init__.py [hunk 1]
```diff
@@ -6,11 +6,14 @@
 
 
 class NameGenerator(generators.Generator):
+    regexp = r'^[a-zA-Z0-9][a-zA-Z0-9_-]*$'
```
Comment (ametaireau) [inline kinto/views/__init__.py]: So we fix it only for kinto and not for all cliquet resources?

**Comment by ametaireau** [comment]
If we want to allow this only for kinto then I think this PR does what it should, but I wonder if we shouldn't do this directly in cliquet.

**Comment by Natim** [comment]
Tests needs to be restarted after mozilla-services/cliquet#411 merge.

**Comment by leplatrem** [comment]
I don't see any objection in (re)defining the regexp here, along with the generation in __call__. 
It would be even better IMO

**Comment by Natim** [comment]
Restarting tests now that cliquet PR has been merged.

**Pull request merged**